### PR TITLE
Nwbeacon improvements and regex replacement with ipv4_is_private

### PR DESCRIPTION
**Pull request by ashwin-patil**
Change(s):
   - Network beacon query improvements per review by @oshezaf 
   - regex replacement 

   Reason for Change(s):
   - Query review changes for improvement and logic update

   Version Updated:
   - yes, updated for detections

   Testing Completed:
   - completed

   Checked that the validations are passing and have addressed any issues that are present:
   - completed

### CODE REVIEW

**Review by oshezaf — APPROVED** [review]

**Review by oshezaf — DISMISSED** [review]

File: Detections/CommonSecurityLog/PaloAlto-NetworkBeaconing.yaml [hunk 1]
```diff
@@ -5,14 +5,15 @@ description: |
...
+  http://www.austintaylor.io/detect/beaconing/intrusion/detection/system/command/control/flare/elastic/stack/2017/06/10/detect-beaconing-with-flare-elasticsearch-and-intrusion-detection-systems/
+  https://techcommunity.microsoft.com/t5/microsoft-sentinel-blog/detect-network-beaconing-via-intra-request-time-delta-patterns/ba-p/779586'
 severity: Low
 requiredDataConnectors:
   - connectorId: PaloAltoNetworks
     dataTypes:
       - CommonSecurityLog
 queryFrequency: 1d
-queryPeriod: 1d
+queryPeriod: 2d
```
[the remark this answers is not in the document]
Comment (ashwin-patil) [reply Detections/CommonSecurityLog/PaloAlto-NetworkBeaconing.yaml:16]: Actually query needs to have full day logs to identify the trend hence i have modified query timestamp to startofday and changed query period to 2d to cover it.

**Review by aprakash13 — APPROVED** [review]
Thanks @ashwin-patil

**Comment by vmanojreddy** [comment]
@ashwin-patil Please resolve the conflicts. thanks!!!

**Comment by vmanojreddy** [comment]
@ashwin-patil Please resolve the conflicts. thanks!!!

**Review by aprakash13 — APPROVED** [review]
Thanks.

**Review by aprakash13 — APPROVED** [review]
Thanks Ashwin.

**Pull request merged**